interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +840,23 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+
+<?php if(isset($_GLOBALS['validate_pages']))  :?>
+    <?php foreach($_GLOBALS['validate_pages'] as $validation) :?>
+        <?php if($validation['page_name']==basename(__FILE__)):?>
+            <?php include_once("$srcdir/validation/validation_script.js.php");?>
+            <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/jquery-1.9.1.min.js"></script>
+            <script type="text/javascript">
+                var form = document.getElementsByName("theform");
+                form.id="theform";
+                <!--Use validation script js Validations-->
+                $("form").attr("onclick",'return submitme(<?php echo $GLOBALS['new_validate'] ? 1 : 0;?>,event,"theform",<?php echo $validation['rules'];?>)');
+            </script>
+        <?php endif;?>
+    <?php endforeach;?>
+<?php endif;?>
+
```

Comment:
Why not just make a variable at the top of the script:
``````
$validate_id = "add_edit_issue";
``````
Then not tied to the name (in case duplicates) or whether ends with php or inc. Then also even have option of using same validator in different scripts. Then would use entry in lists for option_id to hold the validate_id and the title to hold the full path/name to the file(this seems like the clearest way for user to know where the file is).
Then by using validate_id, could do:
``````
if ($validate_id) {
  $validate_form = collect_validate($validate_id);
``````
And the collect_validate function would essentially do the same thing your globals code does, except only does it for 1 entry rather than all of them.